documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -310,106 +103,134 @@ Controller services can be configured in three different ways:
- | `controller.zk.connect.security.enable` | false <sup>2</sup> |
- | `controller.zk.connect.security.tls.trustStore.location` | /etc/secrets/client.truststore.jks |
- | `controller.zk.connect.security.tls.trustStore.pwd.location` | /etc/secrets/client.truststore.jks.password |
- | `controller.security.auth.enable` | true |
- | `controller.security.pwdAuthHandler.accountsDb.location` <sup>3</sup> | /etc/secrets/password-auth-handler.inputfile |
- | `controller.security.auth.delegationToken.signingKey.basis | a-secret-value |
+ | `controller.security.tls.enable` | `true` |
+ | `controller.security.tls.server.certificate.location` | `/etc/secrets/controller01-server-cert.crt` |
+ | `controller.security.tls.server.privateKey.location` | `/etc/secrets/controller01-server-key.key` |
+ | `controller.security.tls.trustStore.location` | `/etc/secrets/ca-cert.crt` |
+ | `controller.security.tls.server.keyStore.location` | `/etc/secrets/controller01-server.keystore.jks` |
+ | `controller.security.tls.server.keyStore.pwd.location` | `/etc/secrets/controller01-server.keystore.jks.password` <sup>1</sup> |
+ | `controller.zk.connect.security.enable` | `false` <sup>2</sup> |
+ | `controller.zk.connect.security.tls.trustStore.location` | Unspecified <sup>2</sup>|
+ | `controller.zk.connect.security.tls.trustStore.pwd.location` | Unspecified <sup>2</sup>|
+ | `controller.security.auth.enable` | `true` |
+ | `controller.security.pwdAuthHandler.accountsDb.location` <sup>3</sup> | `/etc/secrets/password-auth-handler.database` |
+ | `controller.security.auth.delegationToken.signingKey.basis` | `a-secret-value` |
 
  [1]: This and other `.password` files are text files containing the password for the corresponding store.
 
- [2]: It is assumed here that Zookeeper TLS is disabled. You may enable it and specify the corresponding client-side TLS configuration properties via the `controller.zk.*` properties.
+ [2]: It is assumed here that Zookeeper TLS is disabled. You may enable it and specify the corresponding client-side 
+ TLS configuration properties via the `controller.zk.*` properties.
 
  [3]: This configuration property is required when using the default Password Auth Handler only.
 
 **Segment Store**
 
-Segment store supports configuration via a properties file (`config.properties`) or JVM system properties. The table below lists its TLS and auth parameters and sample values. For a detailed discription of these parameters refer to [this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
+Segment store supports security configuration via a properties file (`config.properties`) or JVM system properties. The table 
+below lists its TLS and auth parameters and sample values. For a detailed discription of these parameters refer to
+[this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
```

Comment:
"refer to [this]" -> "refer to the [Pravega Security Configurations]"

Reply:
Done.